src/utils/mongoCollectionManager.js

```diff
@@ -0,0 +1,32 @@
+const Mutex = require('async-mutex').Mutex;
+const mutex = new Mutex();
+
+const {indexCollection} = require('./index.util');
+const {isTrue} = require('./isTrue');
+const env = require('var');
+
+/**
+ * Gets or creates a collection
+ * @param {import('mongodb').Db} db
+ * @param {string} collection_name
+ * @return {Promise<import('mongodb').Collection>}
+ */
+async function getOrCreateCollection(db, collection_name) {
+    if (isTrue(env.CREATE_INDEX_ON_COLLECTION_CREATION)) {
```

Comment:
Should we update the env.JSON file to contain this new variable? And this would be reflected in the helm.helix-service config as well.

Reply:
good catch.  Fixing it

Reply:
done